map_server/src/image_loader.cpp

```diff
@@ -42,6 +42,9 @@
 // We use SDL_image to load the image from disk
 #include <SDL/SDL_image.h>
 
+// Use Bullet's Quaternion object to create one from Euler angles
+#include <LinearMath/btQuaternion.h>
```

Comment:
bullet is not installed on the build machines AFAIK so this will likely fail to compile. We use tf2 quaternions in ROS 2 for now I think